### ISSUE TITLE
Actualize examples

### ISSUE THREAD

**Issue by Alarmfifa** [varspool/Wrench#42]
Hello guys.
Please, actualize your examples.
- wrench/examples/server_ssl.php

It doesn't work, because it tries to register a demo application:
`$server->registerApplication('demo', \Wrench\Application\DemoApplication::getInstance());`
But there are no demo application in the folder:
`$ ls
Application.php  EchoApplication.php  ServerTimeApplication.php`
- wrench/examples/coffeescript/

There are no jsmaker.php (for coffescript convertion) and js files (json2 and jquery). But it isn't main problem. I can convert coffescript's files and add missing js. But it tries to connect to `serverUrl = 'ws://127.0.0.1:8000/demo';`
Could you help me and show a demo server script which I need to launch for this examples?

In summary I didn't find a good working example, therefore the understanding of your websocket realization is extremely difficult =(
